wherehows-web/app/components/datasets/containers/dataset-health.ts

```diff
@@ -52,6 +54,12 @@ export default class DatasetHealthContainer extends Component {
    */
   severityMetrics: Array<IChartDatum> = [];
 
+  /**
+   * Fetched data for the health score detailed data.
+   * @type {Array<IHealthScore>}
+   */
+  tableData: Array<IHealthScore> = [];
```

Comment:
You have `tableData` defined as a class field here, but in the constructor below, you specify a default value, it should be one or the other, the default in the constructor will never enter the default case because the class field will be present. Also, since it's a class field, I'm assuming you don't expect this to be passed in as an external attribute?

Reply:
There shouldn't be a constructor on dataset-health, maybe you're looking at something outdated?

Reply:
👍

Reply:
Scrolled too fast